spotipy/client.py

```diff
@@ -447,22 +508,42 @@ def episodes(self, episodes, market=None):
         tlist = [self._get_id("episode", e) for e in episodes]
         return self._get("episodes/?ids=" + ",".join(tlist), market=market)
 
-    def search(self, q, limit=10, offset=0, type="track", market=None):
+    def search(self, q, limit=10, offset=0, type="track", market=None, total=None):
         """ searches for an item
 
             Parameters:
                 - q - the search query (see how to write a query in the
                       official documentation https://developer.spotify.com/documentation/web-api/reference/search/search/)  # noqa
-                - limit  - the number of items to return (min = 1, default = 10, max = 50)
+                - limit  - the number of items to return (min = 1, default = 10, max = 50). If a search is to be done on multiple
+                            markets, then this limit is applied to each market. (e.g. search US, CA, MX each with a limit of 10).
                 - offset - the index of the first item to return
                 - type - the type of item to return. One of 'artist', 'album',
                          'track', 'playlist', 'show', or 'episode'
                 - market - An ISO 3166-1 alpha-2 country code or the string
-                           from_token.
-        """
-        return self._get(
-            "search", q=q, limit=limit, offset=offset, type=type, market=market
-        )
+                           from_token. Can supply list of markets. Pass "ALL" to search all country codes.
+                - total - the total number of results to return if multiple markets are supplied in the search.
+        """
+        # if string passed and equals "ALL"
```

Comment:
This comment doesn't really tell anything different from the line of code just below it. Let's change it to something like "Search in all available markets" or maybe we could just remove it all together.